csrankings-g.csv

```diff
@@ -539,6 +539,7 @@ Giuseppe M. J. Barca,Australian National University,https://comp.anu.edu.au/peop
 Giuseppe Caire,TU Berlin,https://www.commit.tu-berlin.de/menue/mitarbeiter/fachgebietsleiter,g66ErTcAAAAJ
 Giuseppe Carenini,University of British Columbia,https://www.cs.ubc.ca/~carenini,HNNL22kAAAAJ
 Giuseppe Cattaneo,University of Salerno,http://www.di.unisa.it/~roscigno,PKs7M0gAAAAJ
+Giuseppe Claudio Guarnera,University of York,https://sites.google.com/view/giuseppe-claudio-guarnera,mC7W90kAAAAJ

```

Comment:
Listed with two affiliations: "Senior Researcher at NTNU (Norway) and a Lecturer at the University of York, UK". Is the York affiliation >= 75% time?

Reply:
He is 100% full time at York. The NTNU position is in addition to this.